TEST - CSIP1 test case

koit opened the pull request:
Please review the testCase.xml for CSIP1 `mets/@OBJID`, issue #65. It is compliant with the updated XML template from PR #59. See also E-ARK-CSIP issue https://github.com/DILCISBoard/E-ARK-CSIP/issues/270.

koit committed:
Update testCase.xml for CSIP1 mets/@OBJID

Rewrite to new XML template.
Add rule that @OBJID should equal to package ID/rep ID.

koit committed:
Update OBJID testCase.xml to match latest schema

Implement the schema updates agreed on 2019-02-06, see #72.

koit committed:
Update OBJID testCase.xml

Fix ampersands in references/reference text content.
Fix error in package mets-xml_mets_OBJID_attribute_value_numeric description.
Add valid package descriptions.

PhillipTommerholt commented on corpus/mets-xml/mets/OBJID/testCase.xml:
```diff
@@ -0,0 +1,141 @@
+      | -- | --------------- | ------------------- | ------------------- |
+      | **CSIP1** | **Content Identification** `mets/@OBJID` | It is mandatory to use a content ID which is expressed with @OBJID. The value should be the same as the name or ID of the package (the name of the root folder) for the root METS document or the name and folder name for the representation. The OBJID must meet the principle of being unique at least across the repository. | **1..1** MUST ...
+    </reference>
+    <reference requirementId="the-use-of-identifiers" URL="http://earkcsip.dilcis.eu/#the-use-of-identifiers">
```
I am not sure the ID requirement goes for OBJID. I understand the ID requirement as only for XML-datatype "ID".

PhillipTommerholt reviewed (changes requested):
It is not my understanding that the OBJID should follow the ID xml data type rule.
If I am correct, then the files will have to have several amendments. If I am not correct then it seems fine

koit committed:
Fix OBJID testCase.xml (remove xsd:ID data type)

Remove references to xsd:ID data type, as OBJID is actually xsd:string.

koit committed:
Update OBJID testCase.xml (add valid IP w/ 1 rep)
Add to rule 3 a reference to valid IP valid_minimal_with_one_representation.

koit committed:
Fix OBJID testCase.xml (typo in rule 2 error msg)

koit committed:
Update OBJID testCase.xml to match latest schema
Remove "dependency" element as mentioned in #72.

carlwilson reviewed (commented):
Hi @koit I'm wondering about rule 3. I have a feeling we discussed this and thought that it might be better broken into two rules, one for the main package and the other for representations?

koit committed:
Update OBJID testCase.xml: Split rule 3

Rule 3 (for root METS.xml): OBJID should match package ID.
Rule 4 (for representation METS.xml): OBJID should match representation ID.

koit committed:
Update OBJID testCase.xml to match latest schema

koit committed:
Merge branch 'integration' into csip/mets-root/csip1

Get up to date with the changed folder structure at integration.

koit committed:
Update folder structure

Move testCase.xml to /corpora/csip/metadata/mets-root/CSIP1/.

koit commented:
> Hi @koit I'm wondering about rule 3. I have a feeling we discussed this and thought that it might be better broken into two rules, one for the main package and the other for representations?

I agree @carlwilson, it is done now.
